RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Live.swift

```diff
@@ -401,7 +402,7 @@ extension FactorSourceKind: Comparable {
 
 	fileprivate var signingOrder: Int {
 		switch self {
-		case .ledgerHQHardwareWallet: 0
+		case .ledgerHqHardwareWallet: 0
```

Comment:
unfortunately UniFFI camelcases identifiers so `HQ` -> `Hq`... but since we are switching here, we cannot use a `static let ledgerHQHardwareWallet: Self = . ledgerHqHardwareWallet` to save us in this case (pun intended).